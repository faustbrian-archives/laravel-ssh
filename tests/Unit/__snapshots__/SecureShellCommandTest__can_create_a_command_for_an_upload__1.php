<?php

return 'scp -i /home/root/.ssh/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o BatchMode=yes -o PasswordAuthentication=no -P 22 user@127.0.0.1 /home/root/source:/home/root/target';
