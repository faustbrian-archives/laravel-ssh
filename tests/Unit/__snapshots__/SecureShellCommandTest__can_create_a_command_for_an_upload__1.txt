scp -i /home/root/.ssh/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o BatchMode=yes -o PasswordAuthentication=no -p 22 /home/root/source user@127.0.0.1:/home/root/target